scripts/docker_files/docker_file_ci_ubuntu_bionic/DockerFile

```diff
@@ -46,7 +46,8 @@ RUN apt-get update -y && apt-get upgrade -y && \
     python3 -m pip install --upgrade pip && \
     pip3 install \
         numpy \
-        scipy && \
+        scipy \
+        sympy && \
```

Comment:
this container is not use any more and also not automatically build
Hence I recommend not touching it any more, we will remove it eventually